### ISSUE TITLE
add docs command

### ISSUE THREAD

**Issue by MartinNowak** [D-Programming-Language/dub#501]
Should become the new subcommand for document generation, the current `-b docs` and `-b ddox` methods are hardly configurable.
http://forum.dlang.org/thread/tcunkoqxnzcsjzfajjvn@forum.dlang.org#post-tcunkoqxnzcsjzfajjvn:40forum.dlang.org

**Comment by Geod24** [comment]
+1
The post is a bit old, the unittest-by default is long-fixed.

What should 'docs' be able to do ?
One of the killer feature I'd like to see in is the ability to integrate directly with Github (see: #355 & 449 )

**Comment by John-Colvin** [comment]
Yes please. The lack of configurability is a real pain at the moment

**Comment by Jedzia** [comment]
You should mention in the documentation that the existing -b docs is recursive "dumb". 
I was irritated (several hours long) when i only wanted to generate documentation for my own project.

**Comment by pbbob** [comment]
Desperate ping. I need to build docs for my project only, not dependencies.

**Comment by wilzbach** [comment]
You could add this issue to the DUB User experience story here (https://github.com/dlang/projects/issues/1) and I recommend voting for it (via GitHub reactions).

**Comment by schveiguy** [comment]
My current workaround for this:

1. build using dub -b ddox to get the docs.json file properly generated.
2. remove docs directory
3. run ddox directly with command line I want.

It would be great to be able to configure the doc generation.
